test/markdownlint-cli2-test-cases.js

```diff
@@ -689,6 +689,12 @@ const testCases =
     "usesEnv": true
   });
 
+  testCase({
```

Comment:
Look at pre/post copy/delete directory from other tests.

Reply:
I removed it from here (I think this is was caused the file). The JSON and XML formatter are not listed as a testcase here either.